src/pages/admin/membership-builder/index.vue

```diff
@@ -1,135 +1,166 @@
+      />
+
+      <h4 class="mb-4 text-lg font-semibold">
+        {{ stepT('suggestedAmounts') }} *
+      </h4>
+      <div class="mb-4 flex gap-4">
+        <PeriodAmounts
+          v-for="(period, periodI) in joinContent.periods"
+          :key="period.name"
+          v-model="joinContent.periods[periodI].presetAmounts"
+          :period="period.name"
+          :min-monthly-amount="joinContent.minMonthlyAmount"
+          class="flex-1"
         />
       </div>
+      <div class="mb-4 flex gap-4">
+        <div class="flex-1">
+          <AppLabel :label="stepT('minAmount')" />
+          <AppInput
+            v-model="joinContent.minMonthlyAmount"
+            type="number"
+            :min="1"
+            required
+            class="block w-32"
+          />
+        </div>
+        <div class="flex-1">
+          <AppSelect
+            v-model="selectedDefaultAmount"
+            :label="stepT('defaultAmount')"
+            :items="defaultAmounts"
+            required
+          />
+        </div>
+      </div>
+      <div class="mb-4 flex gap-4">
+        <AppCheckbox
+          v-model="joinContent.showAbsorbFee"
+          :label="stepT('showAbsorbFee')"
+          class="font-semibold"
+        />
+      </div>
+    </AppForm>
+    <div
+      class="bg-cover bg-center p-4 pt-8"
+      :style="`background-image: url(${backgroundUrl})`"
+    >
+      <AuthBox>
+        <JoinForm :join-content="joinContent" @submit.prevent="" />
+      </AuthBox>
     </div>
   </div>
 </template>
 <script lang="ts" setup>
-import mitt from 'mitt';
-import { markRaw, Ref, ref } from 'vue';
-import type { Component } from 'vue';
+import { computed, onBeforeMount, ref } from 'vue';
+import { JoinContent } from '../../../utils/api/api.interface';
+import { fetchContent, updateContent } from '../../../utils/api/content';
+import AppForm from '../../../components/forms/AppForm.vue';
+import AppInput from '../../../components/forms/AppInput.vue';
+import RichTextEditor from '../../../components/rte/RichTextEditor.vue';
+import AppLabel from '../../../components/forms/AppLabel.vue';
 import { useI18n } from 'vue-i18n';
-import PageTitle from '../../../components/PageTitle.vue';
-import AppStepper from '../../../components/stepper/AppStepper.vue';
-import { Step } from '../../../components/stepper/stepper.interface';
-import EditJoinForm from '../../../components/pages/admin/membership-builder/steps/EditJoinForm.vue';
-import AccountConfirmation from '../../../components/pages/admin/membership-builder/steps/AccountConfirmation.vue';
-import IntroMessage from '../../../components/pages/admin/membership-builder/steps/IntroMessage.vue';
-import Emails from '../../../components/pages/admin/membership-builder/steps/EmailsStep.vue';
-import AppButton from '../../../components/forms/AppButton.vue';
-import { MembershipBuilderEmitter } from '../../../components/pages/admin/membership-builder/membership-builder.interface';
```

Comment:
Can you delete this now very happily unused file (membership-builder.interface)?